From: "Ying-Chun Liu (PaulLiu)" <paulliu@debian.org>
Date: Wed, 3 Jun 2026 14:04:15 +0800
Subject: Add version string to configure.ac

Bug-Debian: https://bugs.debian.org/913503
Last-Update: 2018-12-10

The configure.ac uses git to get the version string. However the git command
cannot get the version string because this is not a git repo.
In Debian we should get the string from debian/changelog.
===================================================================
---
 bootstrap | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bootstrap b/bootstrap
index f8de2d3..f44aa27 100755
--- a/bootstrap
+++ b/bootstrap
@@ -5,7 +5,7 @@ set -e
 
 # Generate a VERSION file that is included in the dist tarball to avoid needed git
 # when calling autoreconf in a release tarball.
-git describe --tags --always --dirty > VERSION
+dpkg-parsechangelog | grep Version: | sed -e 's/Version: //' | sed -e 's/-[^-]\+$//' > VERSION
 
 # generate list of source files for use in Makefile.am
 # if you add new source files, you must run ./bootstrap again
